### PR TITLE
[ios]Adjust golden test threshold for TwoPlatformViewsWithOtherBackDropFilterTests based on current macOS version

### PR DESCRIPTION
**Pull request by cyanglaz**
TwoPlatformViewsWithOtherBackDropFilterTests is failing on macOS 13 with the same simulator version used in macOS 12. The image diff looks identical and slightly above the threshold. This PR adjusts the threshold for this test temporarily so our CI can run on both macOS 13 and 12. This change can be reverted when we move all our CI to macOS 13

Fixes https://github.com/flutter/flutter/issues/134740

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making or feature I am adding, or the PR is [test-exempt]. See [testing the engine] for instructions on writing and running engine tests.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/main/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: testing/scenario_app/ios/Scenarios/Scenarios/AppDelegate.m [hunk 1]
```diff
@@ -9,6 +9,18 @@
...
+// A UIViewController that sets YES for its preferedStatusBarHidden property.
+@interface NoStatusBarViewController : UIViewController
+
+@end
+
+@implementation NoStatusBarViewController
+- (BOOL)prefersStatusBarHidden {
+  return YES;
+}
+@end
```
Comment (vashworth) [inline testing/scenario_app/ios/Scenarios/Scenarios/AppDelegate.m:21]: How is this changes to this file relevant?

Comment (cyanglaz) [reply testing/scenario_app/ios/Scenarios/Scenarios/AppDelegate.m:21]: When testing with XCode 15, I found that the statusbar are showing for certain tests. This is not relevant to this particular fix. I can move it to a different PR.

Comment (cyanglaz) [reply testing/scenario_app/ios/Scenarios/Scenarios/AppDelegate.m]: Done

**Review by vashworth — APPROVED** [review]
LGTM

**Pull request merged**